cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -46,10 +46,12 @@
          {{ 'style="background-color:' +
                 value.background_color + ';"'
                 if value.background_color else '' }}>
+
+    {% if value.background_image %}
+            {% set background_image=image(value.background_image, 'original') %}
+    {% endif %}
     <div class="wrapper hero_wrapper"
-          {{ 'style="background-image: ' +
-                value.background_image + ';"'
-                if value.background_image else '' }}>
+         style="background-image: url( {{  background_image.url if background_image else '' }} );">
```

Comment:
Why the structural change from L:50-L:52? If `background_image` isn't set this will end up outputting `style="background-image: url( '' );"`. It should remove the whole inline style if the background image URL is absent.